src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -174,7 +174,7 @@ public void arcadeDrive(double speed, double turn) {
 
   @Override
   public void periodic() {
-    SmartDashboard.putNumber("Navx", navx.getYaw());
+    SmartDashboard.putBoolean("Navx Connectivity", navx.isConnected());
```

Comment:
Can you add another metric of quaternionx+y+z+w?  I think I've seen the isConnected call come back as true when connected to the arduino.